From 5a01e28bd806326b2143e3e6bb28d4780c5d879d Mon Sep 17 00:00:00 2001
From: Victor Kamensky <victor.kamensky7@gmail.com>
Date: Sun, 3 Dec 2023 18:40:05 -0800
Subject: [PATCH] Makefile.am: remove runtime/linux/uprobes and
 runtime/linux/uprobes2 install

"PR30434 continuation:  Removed old uprobes, uprobes2 implementation,
uprobes-inc.h & any mentions of CONFIG_UTRACE." commit removed uprobes,
and uprobes2 sources and directories, but Makefile.am still tries to
install them. In fact after failing to 'cd' into runtime/linux/uprobes
directory it copies top level *.[ch] files into
${prefix}/share/systemtap/runtime/linux/uprobes directory.

The issue was caught by OpenEmbedded project do_package_qa checks.

Signed-off-by: Victor Kamensky <victor.kamensky7@gmail.com>

Upstream-Status: Submitted [https://sourceware.org/pipermail/systemtap/2023q4/027880.html] 
---
 Makefile.am | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 5737c6b20..2ba896088 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -277,10 +277,6 @@ endif
 		do $(INSTALL_DATA) -D $$f $(DESTDIR)$(pkgdatadir)/runtime/softfloat/$$f; done)
 	(cd $(srcdir)/runtime/linux; for f in *.[ch]; \
 		do $(INSTALL_DATA) -D $$f $(DESTDIR)$(pkgdatadir)/runtime/linux/$$f; done)
-	(cd $(srcdir)/runtime/linux/uprobes; for f in Makefile *.[ch]; \
-		do $(INSTALL_DATA) -D $$f $(DESTDIR)$(pkgdatadir)/runtime/linux/uprobes/$$f; done)
-	(cd $(srcdir)/runtime/linux/uprobes2; for f in *.[ch]; \
-		do $(INSTALL_DATA) -D $$f $(DESTDIR)$(pkgdatadir)/runtime/linux/uprobes2/$$f; done)
 	(cd $(srcdir)/tapset; find . \( -name '*.stp' -o -name '*.stpm' -o -name README \) -print \
 		| while read f; do $(INSTALL_DATA) -D $$f $(DESTDIR)$(pkgdatadir)/tapset/$$f; done)
 	(cd $(srcdir)/testsuite/systemtap.examples; find . -type f -print \
-- 
2.31.1

